data/manifest_core.json

```diff
@@ -778,8 +778,12 @@
       },
       "WNV": {
         "resolution": {
+          "global": "",
+          "wa": "",
+          "lineage-1A": "",
+          "lineage-2": "",
           "NA": "",
-          "default": "NA"
+          "default": "global"
```

Comment:
Re: failing checks, since the default changed, this test on the default needs to be updated:

https://github.com/nextstrain/nextstrain.org/blob/2998b8067be993d6ebc7f6f5731c06629ada95e4/test/date_descriptor.test.js#L312

Reply:
Looking at "git blame", @jameshadfield, do we still need to hardcode a redirect from "WNV" to "WNV/NA"? I'm trying to get the context for this date_descriptor check to figure out what to change here.

Reply:
No problem! I think it's fine to rely on PR checks to know when a test should be changed. In this case it required scrolling up more in the [run output](https://github.com/nextstrain/nextstrain.org/actions/runs/13799033821/job/38673874498) to see these lines:

```
FAIL test/inventory_parsing.test.js
  ● core URL redirects (via our manifest) are correctly obtained

    expect(received).toEqual(expected) // deep equality

    Expected: "WNV/NA"
    Received: "WNV/global"
```